webserver/views/user.py

```diff
@@ -55,6 +57,17 @@ def profile(user_name):
 
     user = _get_user(user_name)
 
+    need_exact_listen_count = request.args.get("exact") == 'y'
+    try:
+        have_listen_count = True
+        listen_count = db_conn.get_listen_count_for_user(user_name, need_exact = need_exact_listen_count)
+    except (InfluxDBServerError, InfluxDBClientError):
+        have_listen_count = False
+        listen_count = 0
+
+    if have_listen_count and not need_exact_listen_count and listen_count > LISTEN_COUNT_APPROXIMATION:
+        listen_count = (listen_count / LISTEN_COUNT_APPROXIMATION) * LISTEN_COUNT_APPROXIMATION
```

Comment:
given the python3 discussion, this won't work, since `/` is now float division.
I just learned this:
```python
In [2]: round(4325, -1)
Out[2]: 4330.0
```
from https://docs.python.org/2/library/functions.html#round: "Values are rounded to the closest multiple of 10 to the power minus ndigits;"
so we could do `int(round(count, -1))` - where `-1` is some variable name

Having said this, I'm not sure why we only show an approximation unless the user requests an exact count. Is there a reason why we don't want to be exact all the time?